AutomatedTesting/Gem/PythonTests/Multiplayer/TestSuite_Sandbox.py

```diff
@@ -43,3 +47,26 @@ def test_Multiplayer_BasicConnectivity_Connects(self, request, workspace, editor
     def test_Multiplayer_SimpleNetworkLevelEntity(self, request, workspace, editor, launcher_platform):
         from .tests import Multiplayer_SimpleNetworkLevelEntity as test_module
         self._run_prefab_test(request, workspace, editor, test_module)
+
+    def test_Multiplayer_SimpleGameServerLauncher_ConnectsSuccessfully(self, workspace, launcher_platform):
+        unexpected_lines = []
+        expected_lines = ["New outgoing connection to remote address:"]
+        halt_on_unexpected = False
+        timeout = 180
+
+        # Start the AutomatedTesting.ServerLauncher.exe in hosting mode, no rendering mode, and wait for it to exist
+        server_launcher = launcher_helper.create_server_launcher(workspace)
+        server_launcher.args.extend(['+host', '-rhi=Null'])
+        server_launcher.start()
+        waiter.wait_for(lambda: process_utils.process_exists(f"AutomatedTesting.ServerLauncher.exe", ignore_extensions=True))
+
+        # Start the AutomatedTesting.GameLauncher.exe in client mode, no rendering mode, and wait for it to exist
+        game_launcher = launcher_helper.create_game_launcher(workspace)
+        game_launcher.args.extend(['+connect', '-rhi=Null'])
+        game_launcher.start()
+        waiter.wait_for(lambda: process_utils.process_exists(f"AutomatedTesting.GameLauncher.exe", ignore_extensions=True))
+
+        # Verify that the GameLauncher.exe was able to connect to the ServerLauncher.exe by checking the logs
+        game_launcher_log_file = os.path.join(game_launcher.workspace.paths.project_log(), 'Game.log')
+        game_launcher_log_monitor = LogMonitor(game_launcher, game_launcher_log_file)
+        game_launcher_log_monitor.monitor_log_for_lines(expected_lines, unexpected_lines, halt_on_unexpected, timeout)
```

Comment:
How thorough has this been tested on local/AR? I'm concerned that logging issues may be intermittent and will fail this check.

Reply:
AR currently will not execute these tests, as they are marked for the Nightly Sandbox suite. Being in Sandbox also means there will be no automated notification of failure, and implies that the SIG-Network will be monitoring the health of these tests until they are ready for the Periodic suite.